packages/neos-ui/src/Containers/PrimaryToolbar/DimensionSwitcher/DimensionSelectorOption.js

```diff
@@ -3,24 +3,47 @@ import PropTypes from 'prop-types';
 import style from './style.module.css';
 // eslint-disable-next-line camelcase
 import SelectBox_Option_SingleLine from '@neos-project/react-ui-components/src/SelectBox_Option_SingleLine/index';
+import mergeClassNames from 'classnames';
 
 export default class DimensionSelectorOption extends PureComponent {
     static propTypes = {
         option: PropTypes.shape({
             label: PropTypes.string.isRequired,
-            disallowed: PropTypes.bool
+            disallowed: PropTypes.bool,
+            existing: PropTypes.bool,
```

Comment:
@nezaniel told us that the wording whether a node exists in the desired dimension is "coverage" or "covered"

```suggestion
            covered: PropTypes.bool,
```

Reply:
I adjusted all the variables to use "covered" as naming